appium-tests/android/android.spec.js

```diff
@@ -373,7 +384,6 @@ describe('Camera tests Android.', function () {
                         return driver
                             .waitForElementByXPath('//android.widget.TextView[@text="Gallery"]', 20000)
                             .elementByXPath('//android.widget.TextView[@text="Gallery"]')
-                            .elementByXPath('//android.widget.TextView[@text="Gallery"]')
```

Comment:
I've tried to find any similar issues but with no luck.
Windows: Windows 10 Enterprise, Version 1607, Build 14393.577
Appium: 1.5.0 (maybe even earlier) to 1.6.3
Android SDK Tools (i.e. emulator version) 25.2.2